README.md

```diff
@@ -77,6 +77,11 @@ In order to use compression in HTTP requests you should install
 the  PHP zlib extension. You can use the same install methods as described for the gRPC extension above,
 however most OS` package managers provide a package for the extension.
 
+#### 4.) Install PHP [ext-ffi](https://www.php.net/manual/en/book.ffi.php)
+
+_Experimental_ support for using fibers in PHP 8.1 for Context storage requires the `ffi` extension, and can
+be enabled by setting the `OTEL_PHP_FIBERS_ENABLED` environment variable to a truthy value (`1`, `true`, `on`).
```

Comment:
Should add a comment about preloading for SAPIs besides CLI - to use the fiber support under different SAPIs the binding may have to be preloaded, e.g. by including `vendor/autoload.php` in `opcache.preload`.
(Should consider moving the header file to the toplevel directory if we want to document the usage with `ffi.preload`, but not sure whether that is something we should support/document for now.)

Reply:
I haven't considered that these changes working with other SAPIs. I've updated the readme but will try to add a working example with FPM to verify what actually works.